Update requirements.txt and requirements_dev.txt

milan-tom opened the pull request:
Closes #59 via updating requirements.txt and requirements_dev.txt using `pip-compile --allow-unsafe --generate-hashes <filename>.txt`.

<a href="https://gitpod.io/#https://github.com/Xcov19/covidX/pull/60"><img src="https://gitpod.io/api/apps/github/pbs/github.com/milan-tom/covidX.git/588f0e340eacffcb94da5a2c6b4aa6ed250fc6d5.svg" /></a>

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/xcov19/covidx/60)
<!-- Reviewable:end -->

codecakes commented on requirements_dev.txt:
```diff
@@ -1,10 +1,759 @@
--r requirements.txt
+#
```
there isn't much diff/ between dev and requirements file if it produces a compiled version which includes packages from requirements.

The dev tools keep getting updated and maybe not required for all instances while working on codebase. generally useful only when someone is developing and testing a new feature, they'd dev tools packages.

milan-tom replied:
Okay, I will amend the readme file.

codecakes reviewed (commented):
I see the comment  "[setting up for development](https://github.com/Xcov19/covidX#how-to-setup-for-development)"
it should be changed to just requirements.txt
and maybe another subsection added like if you want to install dev packages the install requirements_dev.txt like `python3 -m pip install -r requirements_dev.txt`

I think the readme should be updated itself;

milan-tom commented:
I have changed th

> I see the comment "[setting up for development](https://github.com/Xcov19/covidX#how-to-setup-for-development)"
> it should be changed to just requirements.txt
> and maybe another subsection added like if you want to install dev packages the install requirements_dev.txt like `python3 -m pip install -r requirements_dev.txt`
> 
> I think the readme should be updated itself;

I have changed and repositioned the command to a position I think makes more sense. I have also added an extra subsection to install developer packages.

codecakes commented on Readme.md:
````diff
@@ -77,6 +84,13 @@ docker run --rm -it -e SECRET_KEY=<YOUR_SECRET_KEY> -e DEBUG_ENV=1 -e CPPFLAGS="
+Setup a virtualenv and run:
+
+```shell script
+CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)" python3 -m pip install -r requirements_dev.txt 
````
`CPPFLAGS="$(pg_config --cppflags)" LDFLAGS="$(pg_config --ldflags)"` this part is not needed for dev tools pkg

milan-tom replied:
I await your approval for the change.

codecakes reviewed (approved):
looks good. there's a slight one liner change i've added. after that pls merge this.
